corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
cool I'll look into this. for now I think https://github.com/dimagi/commcare-hq/commit/019abf57118338abe5b1bae5dff0b152f9682a60 should be sufficient